canonical-web-and-design/jp.ubuntu.com
# reenable redirects with yaml-responses

Make use of https://github.com/canonical-web-and-design/canonicalwebteam.yaml-responses/ to re-enable redirects.

This involves converting `redirects.json` to `redirects.yaml` - and I've also rolled in the change from https://github.com/canonical-web-and-design/jp.ubuntu.com/pull/141.

Fixes #142
Closes #141

QA
--

`./run`

Check redirects work:

- http://0.0.0.0:8012/engage -> http://0.0.0.0:8012/
- http://0.0.0.0:8012/support -> http://0.0.0.0:8012/enterprise-support
- http://0.0.0.0:8012/enterprise-support/plans-and-pricing -> https://www.ubuntu.com/pricing
